README.md

```diff
@@ -1,5 +1,9 @@
 # Will.IAM
 
+[![Build Status](https://travis-ci.org/topfreegames/Will.IAM.svg?branch=master)](https://travis-ci.org/topfreegames/Will.IAM.svg?branch=master)
```

Comment:
Are these the link targets we want?

* Build Status = links to the build status image - link to https://travis-ci.org/topfreegames/Will.IAM instead?
* Coverage Status = links to coveralls (might be OK?)
* Maintainability = links to codeclimate (seems OK)

Reply:
Good catch! Thanks 👍